### DISCUSSION TITLE
Where can I change the font settings of the search results?

### DISCUSSION THREAD

**Discussion by WS64** [koreader/koreader#7178] [Q&A] [answered]
The search results are very hard to read for me since they are very small. I can't find a place to change the settings for these, is there any?

**Comment by pazos** [comment, marked as answer]
Nope. There's no setting to change the font. You'll need to edit the plugin yourself.

Search for `cface` in https://github.com/koreader/koreader/blob/master/plugins/calibre.koplugin/search.lua. You can replace `smallinfofont` with anything that `Font:getFace` understands. Some examples in https://github.com/koreader/koreader/search?q=getFace
